src/integration.ts

```diff
@@ -418,6 +427,158 @@ export class Integration {
       ),
     );
 
+    const subscribeMethod = makePublicMethod(
+      this.ids.subscribeMethod,
+      makeParams({
+        request: f.createTypeReferenceNode("K"),
+        params: f.createIndexedAccessTypeNode(
+          f.createTypeReferenceNode(this.ids.inputInterface),
+          f.createTypeReferenceNode("K"),
+        ),
+      }),
+      f.createBlock([
+        makeConst(
+          this.ids.pathParameter,
+          f.createAsExpression(
+            f.createElementAccessExpression(
+              makePropCall(this.ids.requestParameter, propOf<string>("split"), [
+                f.createRegularExpressionLiteral("/ (.+)/"), // split once
+                f.createNumericLiteral(2), // excludes third empty element
+              ]),
+              f.createNumericLiteral(1),
+            ),
+            f.createTypeReferenceNode(this.ids.pathType),
+          ),
+        ),
+        makeConst(
+          this.ids.sourceConst,
+          makeNew(
+            f.createIdentifier("EventSource"),
```

Comment:
this one is flagged in Node at the moment or only available via `undici` runtime.
constraints can not be established at the moment, but it's worth it to make a "todo" comment for the fu